pages/mod/_id/edit.vue

```diff
@@ -341,46 +345,47 @@ export default {
         ])
       ).map((it) => it.data)
 
-      mod.license = {
-        short: mod.license.id,
-        name: mod.license.name,
-        url: mod.license.url,
+      savedMod.license = {
+        short: savedMod.license.id,
+        name: savedMod.license.name,
+        url: savedMod.license.url,
       }
 
-      if (mod.body_url && !mod.body) {
-        mod.body = (await data.$axios.get(mod.body_url)).data
+      if (savedMod.body_url && !savedMod.body) {
+        savedMod.body = (await data.$axios.get(savedMod.body_url)).data
       }
 
+      /*
       const donationPlatforms = []
       const donationLinks = []
 
-      if (mod.donation_urls) {
-        for (const platform of mod.donation_urls) {
+      if (savedMod.donation_urls) {
+        for (const platform of savedMod.donation_urls) {
           donationPlatforms.push({
             short: platform.id,
             name: platform.platform,
           })
           donationLinks.push(platform.url)
         }
       }
+      */
 
       availableLicenses.sort((a, b) => a.name.localeCompare(b.name))
       return {
-        mod,
-        clientSideType: mod.client_side.charAt(0) + mod.client_side.slice(1),
-        serverSideType: mod.server_side.charAt(0) + mod.server_side.slice(1),
+        savedMod,
+        mod: { ...savedMod },
```

Comment:
This isn't a deep copy?
I think this could cause issues in production usage.

Reply:
It has to be a deep copy to work. `savedMod` and `mod` are kept separate until the user clicks `Save`. Ideally, the `savedMod` would be saved on Labrinth or LocalStorage, so changes persist closing tabs.